package.json

```diff
@@ -58,6 +58,10 @@
     "seed-offerings": "npm run _start -- dist/seed-offerings.js",
     "server": "npm run _start -- dist/main.js",
     "test": "rimraf dist/tests && npm run compile && node --test",
-    "try": "npm run _debug -- dist/client-test.js"
+    "try": "npm run _debug -- dist/client-test.js",
+    "docker:up": "docker-compose up -d",
+    "docker:down": "docker-compose down",
+    "docker:logs": "docker-compose logs -f",
+    "docker:build": "docker-compose build"
```

Comment:
This is great! I'd probably add a `docker-compose down -v` which wipes down all the data too. Maybe under `docker:down-wipe` (or whatever name you have for it because I'm horrible at this! :P )